app/components/moderation-list-row/component.js

```diff
@@ -13,22 +13,42 @@ const ACTION_LABELS = Object.freeze({
         ltDay: 'components.moderation-list-row.submission.submitted',
     },
     [ACCEPTED]: {
-        gtDay: 'components.moderation-list-row.submission.was_accepted_on',
-        ltDay: 'components.moderation-list-row.submission.was_accepted',
+        gtDay: 'components.moderation-list-row.submission.accepted_on',
+        ltDay: 'components.moderation-list-row.submission.accepted',
+        gtDay_automatic: 'components.moderation-list-row.submission.accepted_automatically_on',
+        ltDay_automatic: 'components.moderation-list-row.submission.accepted_automatically',
     },
     [REJECTED]: {
-        gtDay: 'components.moderation-list-row.submission.was_rejected_on',
-        ltDay: 'components.moderation-list-row.submission.was_rejected',
+        gtDay: 'components.moderation-list-row.submission.rejected_on',
+        ltDay: 'components.moderation-list-row.submission.rejected',
     },
 });
 
 
 export default Component.extend({
     theme: service(),
+    i18n: service(),
+
+    noActions: false,
 
     classNames: ['moderation-list-row'],
 
-    dataLoading: computed.not('firstContributors.length'),
+    contributorLoading: computed.not('firstContributors.length'),
+
+    moderatorLoading: computed.none('lastActionCreator'),
+
+    lastActionCreator: computed.alias('latestAction.creator.fullName'),
+
+    latestAction: computed('submission.actions.[]', function() {
```

Comment:
Instead of [copy/pasting code](https://github.com/sheriefvt/ember-osf-reviews/blob/b7fd34405504c08359bb603b69bdf85845a1ccf2/app/components/preprint-status-banner/component.js#L149), maybe move it into a util that can be used in both places.